pkg/apis/v2v/v1beta1/vmimportconfig_types.go

```diff
@@ -25,6 +25,33 @@ import (
 // VMImportConfigSpec defines the desired state of VMImportConfig
 type VMImportConfigSpec struct {
 	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
+
+	// Rules on which nodes controller pod(s) will be scheduled
+	Infra NodePlacement `json:"infra,omitempty"`
```

Comment:
should this field be mandatory ?
Also, what 'Infra' stands for in this context ? perhaps just 'Placement' or 'NodePlacement' which describes the nature of this field as is ?

Reply:
As @jakub-dzon is working on extracting common code from both CDI and vm-import I used the same name of the type as CDI to make it easy to extract.